src/components/MetadataTable/MetadataTable.js

```diff
@@ -203,33 +205,35 @@ export function MetadataTable(props) {
   /**
    * Callback for downloading the table as a CSV file.
    */
-  const DownloadCSV = () => {
-    // todo: should infer the variable names from the table
-    const csvString = convertTableToCSV(presentDS);
-
-    const blob = new Blob([csvString], { type: 'text/csv' });
-    const url = URL.createObjectURL(blob);
-    const a = document.createElement('a');
-    a.setAttribute('hidden', '');
-    a.setAttribute('href', url);
-    a.setAttribute('download', `metadata_${currentTableName.toLowerCase()}_table.csv`);
-    document.body.appendChild(a);
-    a.click();
-    document.body.removeChild(a);
-  };
+  // const DownloadCSV = () => {
+  //   // todo: should infer the variable names from the table
+  //   const csvString = convertTableToCSV(presentDS);
+
+  //   const blob = new Blob([csvString], { type: 'text/csv' });
+  //   const url = URL.createObjectURL(blob);
+  //   const a = document.createElement('a');
+  //   a.setAttribute('hidden', '');
+  //   a.setAttribute('href', url);
+  //   a.setAttribute('download', `metadata_${currentTableName.toLowerCase()}_table.csv`);
+  //   document.body.appendChild(a);
+  //   a.click();
+  //   document.body.removeChild(a);
+  // };
 
   const postTableData = () => {
 
+
     // this will be rewritten as a web socket eventually as its nice to have two way communication
     // also it is insane to post the entire table every time a single value is changed but YOLO
     let data = { 'table': tables, 'user': props.user["http://schema.org/alternateName"], 'project': props.project };
+    
+    var target = '/writeTable'
+
     if (process.env.NODE_ENV === "development") {
       var target_url = process.env.REACT_APP_OIDC_CLIENT_REDIRECT_URL;
-      var target = `${target_url}/writeTable`
-    }
-    else {
-      var target = '/writeTable'
+      target = `${target_url}/writeTable`
```

Comment:
this should use your target variable as the second half of the URL

Reply:
See 61073166d6a5dd9d564ebb56195067aac87837b4